schema/dmt/schema.go

```diff
@@ -155,11 +160,11 @@ func init() {
 		[]schema.StructField{},
 		schema.StructRepresentation_Map{},
 	))
-	ts.Accumulate(schema.SpawnStruct("TypeList",
+	ts.Accumulate(schema.SpawnStruct("TypeDefnList",
 		[]schema.StructField{
-			schema.SpawnStructField("valueType", "TypeTerm", false, false),
-			schema.SpawnStructField("valueNullable", "Bool", false, false), // TODO: wants to use the "implicit" feature, but not supported yet
-			schema.SpawnStructField("representation", "ListRepresentation", false, false),
+			schema.SpawnStructField("valueType", "TypeNameOrInlineDefn", false, false),
+			schema.SpawnStructField("valueNullable", "Bool", true, false),                // TODO: wants to use the "implicit" feature, but not supported yet
+			schema.SpawnStructField("representation", "ListRepresentation", true, false), // XXXXXX
```

Comment:
FYI I marked all the places where I did implicit -> optional with XXXXX. Not super important given we'll nuke this file ASAP.